.github/workflows/create-release-branch.yaml

```diff
@@ -0,0 +1,28 @@
+name: Create release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersionNumber:
+        description: 'New version number'
+        required: true
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: fregante/setup-git-user@v1
```

Comment:
I've not used this, but have used peter-evans/create-pull-request@v4 which will commit AND then create the PR too. It also picks a random branch name to avoid any conflicts.

What do you think of that?

Reply:
Yeah, done! It results in this: https://github.com/greg-finley/sqlfluff/pull/7

Apologies for the dirty commit history; I have to mess up my fork's `main` branch to get actions to run. Luckily it will get squashed.